src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/FormHandler.php

```diff
@@ -23,7 +23,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-namespace PrestaShopBundle\Form\Admin\AdvancedParameters\Administration;
+namespace PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\Administration;
```

Comment:
gosh, the pull request about Form Handling is still not merged, this making me so crazy :/